api/can_api_v2_definition.py

```diff
@@ -121,6 +134,45 @@ class ActualsTimeseriesRow(Actuals):
     date: datetime.date = pydantic.Field(..., description="Date of timeseries data point")
 
 
+class AnomalyAnnotation(base_model.APIBaseModel):
+    date: datetime.date = pydantic.Field(..., description="Date of anomaly")
+    description: str
+
+
+class MetricAnnotations(base_model.APIBaseModel):
+    """Annotations associated with one field."""
+
+    sources: List[MetricSource]
+    anomalies: List[AnomalyAnnotation]
+
+
+class Annotations(base_model.APIBaseModel):
+    """Annotations for each field."""
+
+    cases: Optional[MetricAnnotations] = pydantic.Field(None, description="Annotations for cases")
+    deaths: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for deaths",
+    )
+    positiveTests: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for positiveTests"
+    )
+    negativeTests: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for negativeTests"
+    )
+    contactTracers: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for contactTracers"
+    )
+    hospitalBeds: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for hospitalBeds"
+    )
+    icuBeds: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for icuBeds"
+    )
+    newCases: Optional[MetricAnnotations] = pydantic.Field(
+        None, description="Annotations for newCases",
+    )
```

Comment:
Where did this list of 8 fields come from?  It seems like a subset of the fields in our API...?

Reply:
From Actuals. Added comment in 41a560c8 to keep them in sync. We could add annotations for metrics too, but I haven't tested that code path. From a glance I think only test positivity will have them.